### PR TITLE
Fix OSD speed unit preview

### PR DESCRIPTION
**Pull request by McGiverGim**
The GPS Speed OSD element preview is wrong, depending on the units.
It is showing `km/h` for British, and `mp/h` for Imperial and Metric.

This PR changes this, using `km/h` for Metric, and `mp/h` for Imperial and British, like in the firmware part:
https://github.com/betaflight/betaflight/blob/025ee87a7aca068e3659fd066b8a9afbed123361/src/main/osd/osd_elements.c#L492-L505

### CODE REVIEW

**Comment by haslinghuis** [comment]
Have implemented #2146 on user request. The user was from GB and insisted on using km for GPS speed. But it looks like British people are using one or the other depending on birthday?

**Comment by McGiverGim** [comment]
The actual code was wrong indeed. It was showing bad the units for Metric.
Now we can discuss if British/Imperial must shown one or another and I can modify the PR, but I have applied the same than firmware if I've looked at the correct code. If something is wrong then it must be changed on firmware first.

**Review by haslinghuis — APPROVED** [review]
Compared with firmware and you are indeed correct.
My bad it was one of my first code PR's :smiley:

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> PASS
 - PR is not in draft -> PASS
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed (3/06/2021 21:16:16) -> PASS
 - commit count less or equal to three (1 commits) -> PASS
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (1 approvers) -> FAIL

**Review by limonspb — APPROVED** [review]

**Review by mikeller — APPROVED** [review]

**Pull request merged**
